packages/extension/src/view/devtools/components/privateAdvertising/protectedAudience/adUnits/adMatrix.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies.
+ */
+import React from 'react';
+import { MatrixComponent } from '@google-psat/design-system';
+
+const AdMatrix = () => {
+  const matrixData = [
+    {
+      title: 'Ad Units',
+      count: 5,
+      color: '#5CC971',
+      description: 'Placeholder',
+      countClassName: 'text-[#5CC971]',
+    },
+    {
+      title: 'Bidders',
+      count: 10,
+      color: '#F3AE4E',
+      description: 'Placeholder',
+      countClassName: 'text-[#F3AE4E]',
+    },
+    {
+      title: 'Bids',
+      count: 15,
+      color: '#4C79F4',
+      description: 'Placeholder',
+      countClassName: 'text-[#4C79F4]',
+    },
+    {
+      title: 'No Bids',
+      count: 20,
+      color: '#EC7159',
+      description: 'Placeholder',
+      countClassName: 'text-[#EC7159]',
```

Comment:
Colours for these are present in the `tailwind.config.cjs`. Maybe we can change that colour's name to an actual colour and reuse them here?